src/application-base/ApplicationBase.jsx

```diff
@@ -38,36 +31,10 @@ const propTypes = {
    * If the `locale` prop is not provided, the preferred language from the browser will be used.
    */
   locale: PropTypes.string,
-  /**
-   * Custom translations for the current locale.
-   */
-  customTranslatedMessages: (props, propName, componentName) => {
-    if (!props[propName]) {
-      return null;
-    }
-
-    if (Object.keys(props[propName]).length !== 0 && props.locale === undefined) {
-      return new Error(`Missing locale prop for ${propName} in ${componentName} props`);
-    }
-
-    return null;
-  },
-  /**
-   * The component to render while the translation files are being retrieved.
-   * NOTE: Absolutely no locale-dependent logic should be
-   * utilized in this placeholder.
-   */
-  translationsLoadingPlaceholder: PropTypes.node,
   /**
    * The name of the theme to apply to the application using terra-theme-provider.
    */
   themeName: PropTypes.string,
```

Comment:
I kind of wish this theme-name related to the name of the theme and not the css class to apply and application base would be smart enough to set the theme class to null if the lowlight theme is the default theme and you pass in the lowlight name. Right now you'd get the lowlight class name applied regardless. It still works fine, but kind of annoys me.